addons/kafka/1.x/kafka-2.yaml

```diff
@@ -15,6 +15,8 @@ spec:
   requires:
     - matchLabels:
         kubeaddons.mesosphere.io/name: zookeeper
+    - matchLabels:
```

Comment:
Since this significantly limits how one can use and test Kafka, should we add a comment here explaining the "why" to any future readers?

Reply:
yeah, let me add
KUDO Kafka monitoring relies on the `prometheus` operator and uses objects like service monitor.
In case this is an `enterprise` addon with possible support. We wouldn't recommend or provide a way for users to install KUDO Kafka without monitoring which means without `prometheus` installed. As this would limit considerably any effort to effectively monitor KUDO Kafka clusters.

Reply:
:+1:  check this comment out for spark too, for possible relevance:
https://github.com/mesosphere/kubeaddons-enterprise/pull/18#discussion_r374128574